eventbridge.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_cloudwatch_event_rule" "aws_backup" {
```

Comment:
@jason-drouhard - Does this really only detect backup changes? I would think that it would also detect if the Environment tag was not added for instance. Is there a way to make it specific to a tag, or would that require a lambda or similar?

Reply:
That's a really good question and one I would need to look into. In the mean time, maybe i should make the name "required-tags"? When I think about it, that name makes more sense since the EventBridge rule triggers if the "required-tags" config rule is noncompliant.

Reply:
I think that makes more sense.